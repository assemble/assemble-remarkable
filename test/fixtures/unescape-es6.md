# es6 templates

${foo}
${foo.bar}
${"foo.bar"}