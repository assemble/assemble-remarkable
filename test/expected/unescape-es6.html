<h1>es6 templates</h1>
<p>${foo}
${foo.bar}
${"foo.bar"}</p>
